src/views/ScheduledChargeConfig.vue

```diff
@@ -41,6 +41,47 @@
 					</openwb-base-button-group-input>
 				</div>
 			</openwb-base-card>
+			<openwb-base-card title="Phasenumschaltung bei PV-Überschuss">
+				<div v-if="$store.state.mqtt['openWB/general/extern'] === true">
+					<openwb-base-alert subtype="info">
+						Diese Einstellungen sind nicht verfügbar, solange sich
+						diese openWB im Steuerungsmodus "secondary" befindet.
+					</openwb-base-alert>
+				</div>
+				<div v-else>
+					<openwb-base-button-group-input
+						title="Anzahl Phasen"
+						:buttons="[
+							{ buttonValue: 1, text: '1' },
+							{ buttonValue: 3, text: 'Maximum' },
+							{ buttonValue: 0, text: 'Automatik' },
+						]"
+						:model-value="
+							$store.state.mqtt[
+								'openWB/general/chargemode_config/scheduled_charging/phases_to_use_pv'
+							]
+						"
+						@update:model-value="
+							updateState(
+								'openWB/general/chargemode_config/scheduled_charging/phases_to_use_pv',
+								$event,
+							)
+						"
+					>
+						<template #help>
+							Hier kann eingestellt werden, ob Ladevorgänge im
+							Modus "Zielladen" bei Laden mit PV-Überschuss mit
+							nur einer Phase oder dem möglichen Maximum in
+							Abhängigkeit der "Ladepunkt"- und
+							"Fahrzeug"-Einstellungen durchgeführt werden. Im
+							Modus "Automatik" entscheidet die Regelung, welche
+							Einstellung genutzt wird, um das Ziel zu erreichen.
+							Voraussetzung ist die verbaute Umschaltmöglichkeit
+							zwischen 1- und 3-phasig (s.g. 1p3p).
+						</template>
+					</openwb-base-button-group-input>
+				</div>
+			</openwb-base-card>
```

Comment:
Es gibt ja schon eine Karte "Phasenumschaltung". Inhaltlich passt die neue Einstellung da rein, daher bitte beide Einstellungen in eine Karte.
Die bestehende Einstellung könntest Du in "Anzahl Phasen zum Ziel erreichen" umbenennen.

Reply:
> Es gibt ja schon eine Karte "Phasenumschaltung". Inhaltlich passt die neue Einstellung da rein, daher bitte beide Einstellungen in eine Karte. Die bestehende Einstellung könntest Du in "Anzahl Phasen zum Ziel erreichen" umbenennen.

Wurde angepasst.